src/game_api/drops.hpp

```diff
@@ -14,7 +14,6 @@ struct DropEntry
     uint32_t vtable_rel_offset;
     uint8_t value_offset = 0;      // the offset of the value to be replaced within the pattern
     uint8_t vtable_occurrence = 1; // when a value occurs more than once in the same virtual table function, choose how many times to replace
```

Comment:
can i remove `vtable_occurrence` ?
it's not used and makes the code for replace_drop awful

Reply:
How does it make that code worse exactly?

Reply:
it's just extra code, looks less clear in my opinion

Reply:
But it's more accurate in case this scenario does eventually happen.

Reply:
Isn't that a rule of programming? don't solve problems that don't exist yet

Reply:
Just FYI: This was obviously added because it was needed at the time. The 1.21.0c (i believe) release invalidated 90% of the patterns, and when those were re-implemented, this was simply not used any more in any of the patterns. While I agree with your rule, the past is also a thing. 
Also, why are the `SACRIFICE_*` drops added when there were similar ones as `ALTAR_*` ? Aren't those duplicates?

Reply:
@zappatic  The `ALTAR_*` ones replace what you get, for example: `ALTAR_DICE_ROPEPILE` replaces rope pile
`SACRIFICE_*` replace what you sacrifice (what you put on the altar)